A21-Service-Config-Error-Handling.md

```diff
@@ -63,12 +66,13 @@ configuration.
 A valid service config needs to be well formatted, i.e., the JSON representation
 of the service config needs to be in a valid JSON format. If the service config
 is well formatted, the client validates each field that it recognizes while
-ignoring the unknown fields, i.e., unknown fields do not affect validation. For
-known fields, valid values are defined by the field itself. Fields can accept
-lists where some of the values may be unknown. As long as there is atleast one
-value in the list which succeeds in getting parsed, the field will be
-considered to have a valid value. If any of the known fields fails in
-validation, the entire service config is deemed invalid.
+ignoring the unknown fields, i.e., unknown fields do not affect validation.
+
+Fields in the service config must clearly define the validation criteria for
+values. Fields can accept lists where some of the values may be unknown. As
```

Comment:
Suggest more precise wording: "In a case where different clients may support different values for a field, a gRFC may define a field to be a list of values, where each client will use the first value in the list that it understands."

Reply:
Suggestion accepted!